EIPS/eip-meta-retro.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
```suggestion
---
eip: 7675
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.